test/Azure.ApiHub.Sdk.Tests/app.config

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't need this or the redirect. As part of the migration, let's make sure the references are set to the versions used by the SDK.